modules/oe_whitelabel_extra_project/oe_whitelabel_extra_project.module

```diff
@@ -14,17 +14,30 @@ use Drupal\media\Plugin\media\Source\OEmbed;
 use Drupal\oe_bootstrap_theme\ValueObject\ImageValueObject;
 use Drupal\Component\Utility\Html;
 
-/**
- * Implements hook_preprocess().
- */
-function oe_whitelabel_extra_project_preprocess_pattern_description_list__field_group__group_info__oe_organisation(array &$variables): void {
-  return;
-}
 /**
  * Implements hook_preprocess().
  */
 function oe_whitelabel_extra_project_preprocess_pattern_description_list(array &$variables): void {
-  return;
+  /** @var \Drupal\ui_patterns\Element\PatternContext $context */
+  $context = $variables['context'];
+  if ($context->getType() === 'field_group') {
+    $id = $context->getProperty('entity_type')
+      . '.' . $context->getProperty('bundle')
+      . '.' . $context->getProperty('view_mode')
+      . '.' . $context->getProperty('group_name');
+    switch ($id) {
+      case 'oe_organisation.oe_cx_project_stakeholder.default.group_info':
+      case 'node.oe_project.full.group_project_details':
+      case 'node.oe_project.full.group_coordinators':
+      case 'node.oe_project.full.group_period':
+      case 'node.oe_project.full.group_budget':
+      case 'node.oe_project.full.group_website':
+        /** @var \Drupal\Core\Template\Attribute $attributes */
+        $attributes = $variables['attributes'];
+        $attributes->addClass('border-bottom');
+        break;
+    }
+  }
```

Comment:
To be clear: I don't like this solution at all!
On the one hand, we currently need these lines under every such field group.
On the other hand, it seems wrong to make this the default behavior.

Reply:
we decided we keep it